webrender/res/ps_line.fs.glsl

```diff
@@ -0,0 +1,123 @@
+#line 1
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+float det(vec2 a, vec2 b) {
```

Comment:
It's a 2d cross product, I guess. It's just using the same name as the paper the algorithm was taken from.